src/bindings/python/src/pyopenvino/frontend/onnx/extension.cpp

```diff
@@ -41,3 +42,37 @@ void regclass_frontend_onnx_ConversionExtension(py::module m) {
 
     ext.def_property_readonly_static("m_converter", &ConversionExtension::get_converter);
 }
+
+void regclass_frontend_onnx_OpExtension(py::module m) {
+    py::class_<OpExtension<void>, std::shared_ptr<OpExtension<void>>, ConversionExtension> ext(
```

Comment:
OpExtension::Ptr?

Reply:
we don't have default Ptr for onnx::OpExtension, we can use onnx::OpExtension with different template types